src/RelayServer.ts

```diff
@@ -64,6 +64,8 @@ import {
 } from './relayServerUtils';
 import { getPastEventsForHub } from './getPastEventsForHub';
 import type { PastEventOptions } from './definitions';
+import { EVENT_REPLENISH_CHECK_REQUIRED } from './events/checkReplenish';
```

Comment:
I would import the `EVENT_REPLENISH_CHECK_REQUIRED` from the index, instead of the file.

Reply:
Imported the event name from the `events` module, thank you